_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
+- " ``rootpath`` determines root directory for input data."
+- " ``output_dir`` is the destination directory."
 ---
 
 ## The configuration file
 
 The ``config-user.yml`` configuration file contains all the global level information needed by ESMValTool to run.
-This is an (YAML file) [https://yaml.org/spec/1.2/spec.html]. An example configuration file can be found in the root directory of the ESMValTool repository.
-Make a copy and rename it to ``config-user.yml``:
+This is an [YAML file](https://yaml.org/spec/1.2/spec.html). An example configuration file can be found in the root directory of the ESMValTool repository: [config-user-example.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/config-user-example.yml).
 
+Let's download it to our working directory ``esmvaltool_tutorial`` that is made during the [Setup](https://esmvalgroup.github.io/tutorial/setup.html).
+To do that, click on [this link](https://raw.githubusercontent.com/ESMValGroup/ESMValTool/master/config-user-example.yml) to see a raw version of the file, right-click and press ``save as``, then you can rename it to ``config-user.yml`` and save it into the working directory ``esmvaltool_tutorial``.
+
+Now in a terminal, let's change our working directory to ``esmvaltool_tutorial``. Then, we run a text editor called Nano to open the configuration file:
+
+~~~bash
+  cd esmvaltool_tutorial
+  nano config-user.yml
 ~~~
-  cp config-user-example.yml config-user.yml
-~~~
-{: .source}
 
 This file contains the information for:
-  * Rootpaths to the data from different projects
-  * Directory structure for input data
-  * Number of available CPUs
-  * Destination directory
-  * Auxiliary data directory
-  * Output settings
-
-## Rootpaths to input data
-ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source, like CMIP for dataset from climate model intercomparison project, and OBS for observational dataset that adhere to (CMOR standard)[https://cmor.llnl.gov/].
-For each category, you can define either one path or several pathes as a list.
-In this lesson, you work with data from (CMIP5)[https://esgf-node.llnl.gov/projects/cmip5/].
-Add the root path of the folder where you downloaded the data during the (Setup)[https://escience-academy.github.io/lesson-esmvaltool/setup.html].
 
+* Rootpath to input data
+* Directory structure for the data from different projects
+* Number of parallel tasks
+* Destination directory
+* Auxiliary data directory
+* Output settings
+
+> ## Which text editor
+>
+> No matter what editor you use, you will need to know where it searches for and saves files. If you start it from the shell, it will (probably) use your current working directory as its default location. We use ``nano`` in examples because it is one of the least complex text editors. Press <kbd>ctrl</kbd> + <kbd>O</kbd> to save the file, and then <kbd>ctrl</kbd> + <kbd>X</kbd> to exit ``nano``.
+{: .callout}
```

Comment:
This command is still visible in the text in it's formatted form. Shouldn't be, right?

Reply:
It is shown in the correct style on the webpage. Could you please check it via [previewing your changes locally](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md#previewing-your-changes-locally)

Reply:
Sorry, I have trouble today with everything, as it seems. Can't get it to work. Maybe @LisaBock can have a look if it is displayed correctly?

Reply:
Yes, looks correct.